android-components/components/feature/fxsuggest/src/main/java/mozilla/components/facts/FxSuggestFacts.kt

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.fxsuggest.facts
```

Comment:
Ooop, sorry I didn't notice this earlier!

This file's location is `android-components/components/feature/fxsuggest/src/main/java/mozilla/components/facts/FxSuggestFacts.kt`...

But the package name is `mozilla.components.feature.fxsuggest.facts`, and Android Studio is warning that the package name doesn't match the file location.

Would you mind moving it to `android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/facts/FxSuggestFacts.kt`, please? 😅

Reply:
okie doke, moved.